src/components/ToolMenu/Draw/index.tsx

```diff
@@ -317,6 +318,7 @@ export const Draw: React.FC<DrawProps> = ({
           </span>
         </DeleteButton>
       ) : <></>}
+      <ColorPickerDrawer />
```

Comment:
The icons looks a litte bit different than the others in the menu (dark background). Probably the `type` needs to be set to `link` as well?